test/docker-compose.yml

```diff
@@ -36,10 +37,15 @@ services:
         volumes:
             - ${PWD}/vna_config/:/run/secrets:ro
         healthcheck:
-            test: ["CMD", "curl", "-f", "-u", "orthanc:orthanc", "http://vna-qr:8042/patients"]
-            interval: 10s
-            timeout: 30s
-            retries: 5
+            test:
+                [
+                    "CMD-SHELL",
+                    "/probes/test-aliveness.py --user=orthanc --pwd=orthanc",
+                ]
+            start_period: 10s
+            retries: 2
+            interval: 3s
+            timeout: 2s
```

Comment:
Has this been tested on the GAE?
I investigated the use of the `start_` tags, but IIRC they only work on docker engine >=25.0 and the GAE uses 23.0 or something, so I think this will refuse to work on the GAE.